indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -112,81 +160,53 @@
                         </div>
                     {% endif %}
                 </div>
-            </div>
-            <div class="i-box-content js-form-container">
-                {% if review.proposed_action.name == 'accept' %}
-                    {% if review.proposed_contribution_type %}
-                        {% set contribution_type -%}
-                            <strong>{{ review.proposed_contribution_type.name|escape }}</strong>
-                        {%- endset %}
-                        {% trans type=contribution_type|safe -%}
-                            Proposed to accept as {{ type }}
-                        {%- endtrans %}
-                    {% else %}
-                        {% trans -%}
-                            Proposed to accept
+                <div class="i-box-content js-form-container">
+                    {% if review.proposed_action.name == 'change_tracks' %}
+                        {% set tracks = render_tracks(review.proposed_tracks) %}
+                        {% trans count=review.proposed_tracks|length, tracks=tracks -%}
```

Comment:
`tracks=tracks` is not needed, plain variables work fine without specifying them here